xbmc/cores/VideoPlayer/DVDCodecs/Audio/DVDAudioCodecFFmpeg.cpp

```diff
@@ -65,7 +65,7 @@ bool CDVDAudioCodecFFmpeg::Open(CDVDStreamInfo &hints, CDVDCodecOptions &options
       allowdtshddecode = atoi(it->m_value.c_str());
 
   if (hints.codec == AV_CODEC_ID_DTS && allowdtshddecode)
-    pCodec = avcodec_find_decoder_by_name("libdcadec");
```

Comment:
Afaik on pi1 the dvdcodecoptions were set at that time. Let's remove it and see.